roles/calico/templates/calico.service.j2

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=calico
+After=docker.service etcd.service
+Requires=docker.service etcd.service
+
+[Service]
+Type=forking
+RemainAfterExit=yes
+Restart=on-failure
+RestartSec=20
+TimeoutStartSec=20m
+
+ExecStartPre=-/usr/bin/docker kill calico-node
+ExecStartPre=-/usr/bin/docker rm calico-node
+Environment="ETCD_AUTHORITY={{ calico_etcdaddr }}"
+ExecStart=/usr/bin/calicoctl node --ip={{ ansible_default_ipv4.address }} \
+                                  --node-image={{ calico_image }}:{{ calico_image_tag }}
+
+ExecStop=/usr/bin/calicoctl node stop
```

Comment:
if we're killing and removing existing nodes before start, shouldn't we kill them at stop as well to make sure everything's clean? (`docker rm -f {name}` works for this in one command, BTW)